woudc_data_registry/processing.py

```diff
@@ -43,21 +43,31 @@
 #
 # =================================================================
 
-from datetime import datetime
-import logging
+
 import os
+import yaml
+
 import shutil
+import logging
+
+from datetime import datetime
 
 from woudc_data_registry import config, registry, search
 from woudc_data_registry.models import (Contributor, DataRecord, Dataset,
                                         Deployment, Instrument, Project,
-                                        Station)
-from woudc_data_registry.parser import (ExtendedCSV, MetadataValidationError,
+                                        Station, StationName)
+from woudc_data_registry.parser import (PROJECT_ROOT, DOMAINS, ExtendedCSV,
```

Comment:
As discussed, we should remove the `PROJECT_ROOT` pathing from `woudc_data_registry.parser`